tests/Unit/Domain/Structure/Test_ElementId.cpp

```diff
@@ -171,6 +171,14 @@ void test_element_id() {
   CHECK(ElementId<3>::external_boundary_id().segment_ids() ==
         make_array<3>(SegmentId(ElementId<3>::max_refinement_level - 1, 0)));
   CHECK(ElementId<3>::external_boundary_id().grid_index() == 0);
+
+  ElementId<3> zeroth_element(0);
+  ElementId<3> not_zeroth_element1{0, 1};
+  ElementId<3> not_zeroth_element2{0, {{{1, 0}, {2, 1}, {1, 0}}}, 0};
+
```

Comment:
add more checks with:
- not on block 0
- the zeroth_element is true, but the `ElementId` is not the root element of block 0